app/src/components/adminConsole/manageTeam/SnappablesTable.tsx

```diff
@@ -1,35 +1,110 @@
 import React from "react";
 import { Entity, Snap, Snappable } from "../../../types";
-import SnappableRow from "./SnappablesRow";
 import { countSnapsForUser } from "./CountSnapsForUser";
+import { useTable, useSortBy, useFlexLayout } from "react-table";
 
 const SnappablesTable = (props: {
     snappables: Entity<Snappable>[];
     currentSnaps?: Entity<Snap>[];
 }) => {
-    const rows = props.snappables
-        .sort((a: Snappable, b: Snappable) =>
-            a.fullName.localeCompare(b.fullName)
-        )
-        .map((p: Snappable) => (
-            <SnappableRow
-                key={p.id}
-                snappable={p}
-                numSnaps={countSnapsForUser(p.id, props.currentSnaps)}
-            />
-        ));
+    const defaultColumn = React.useMemo(
```

Comment:
Is `useMemo` necessary here, and below, given these are just constants?

Reply:
The react table docs explicitly demand the use of `useMemo` for anything that goes as an input to the `useTable` hook otherwise we get into an infinite loop of re-renders. I have checked and it does indeed error if this isn't memoised.